benchmarks/nuts/hlr.jl

```diff
@@ -0,0 +1,25 @@
+using Turing, TuringBenchmarks.TuringTools
```

Comment:
Kai and I plan to move all the micro-benchmarks into `MCMCBenchmarks`, and depreciate `TuringBenchmarks` together with `ContinousBenchmarks.jl`.  Given that this transition will happen soon, maybe 

- remove the dependency on `TuringBenchmarks.TuringTools`
- and replace `@tbenchmark` with `@benchmark` from `BenchmarkTools`